tests/manage/z_cluster/test_delete_local_volume_sym_link.py

```diff
@@ -0,0 +1,70 @@
+import logging
+import pytest
+
+from ocs_ci.ocs.utils import get_pod_name_by_pattern
+from ocs_ci.ocs.defaults import ROOK_CLUSTER_NAMESPACE
+from ocs_ci.framework.testlib import E2ETest, tier4, tier4a, tier4b, tier4c
+from ocs_ci.ocs import ocp, constants
+from ocs_ci.framework.pytest_customization.marks import skipif_no_lso
+from tests.helpers import wait_for_resource_state
+from ocs_ci.ocs.resources.pvc import get_deviceset_pvcs
+from ocs_ci.ocs.resources.pod import wait_for_storage_pods, get_pod_obj, get_pod_node
+from ocs_ci.framework import config
+from ocs_ci.utility.utils import ceph_health_check
+
+log = logging.getLogger(__name__)
+
+
+@tier4
+@tier4a
+@tier4b
+@tier4c
```

Comment:
I meant you shuld mark only one out of a, b or c

Reply:
What to choose?